en/docs/updates/updates-portal.md

```diff
@@ -0,0 +1,14 @@
+
+WSO2 Update portal is also called 'WSO2 Updates Information Portal'.
+It is UI presentation provisioned to all WSO2 users to visualize a cluster of information.<br>
+Accessing the Updates Portal is easy. It requires you to be a WSO2 user to access the portal .<br>
+Follow the below steps: <br>
+1. Sign in to the [Updates Portal](https://updates-info.wso2.com/). <br>
+2. Enter your WSO2 user credentials and click on the 'Sign In' button. <br>
+3. The page will be directed to the 'WSO2 Updates Information Portal'<br>
+
+Some of the main benefits users would receive by using the Updates Portal are described below:
+
+*   You would receive a comprehensive set of details of the Updates available, already installed updates,and last update date.<br>
+*   Update Levels for a specific product is can be easily searched, and the user is able to refer descriptions and instructions related to specific updates and hotfixes.<br>
```

Comment:
Should be corrected as `Update Levels for a specific product can be easily searched, and the user is able to refer descriptions and instructions related to specific updates ` 
(note the removal of `is` and `and hotfixes`), since we do not provide descriptions and instructions related to hotfixes in the updates info portal